osu.Game.Rulesets.Osu/Replays/OsuAutoGenerator.cs

```diff
@@ -246,6 +246,7 @@ private void moveToHitObject(OsuHitObject h, Vector2 targetPos, Easing easing)
 
             if (waitTime > lastFrame.Time)
             {
+                framesPendingAdjust.Remove(lastFrame);
```

Comment:
this needs an inline comment explaining what it's doing.

Reply:
> this needs an inline comment explaining what it's doing.

```suggestion
                framesPendingAdjust.Remove(lastFrame); // No need to adjust notes that are too far in time
```
Is this Ok?